src/services/codefixes/fixUnusedIdentifier.ts

```diff
@@ -38,14 +47,24 @@ namespace ts.codefix {
                     }
                     break;
                 case fixIdDelete:
-                    tryDeleteDeclaration(changes, sourceFile, token);
+                    if (diag.code === Diagnostics.No_import_of_0_is_used.code) {
+                        changes.deleteNode(sourceFile, getImportDeclarationAtErrorLocation(token));

```

Comment:
Which adjacent trivia does this delete?

Reply:
Looks like it removes leading but not trailing trivia by default.